--- a/usr/lib/llvm-3.7/include/llvm/IR/ValueMap.h	2023-05-25 22:47:21.103192824 +0000
+++ b/usr/lib/llvm-3.7/include/llvm/IR/ValueMap.h	2023-05-25 22:46:58.357808722 +0000
@@ -99,7 +99,7 @@
   explicit ValueMap(const ExtraData &Data, unsigned NumInitBuckets = 64)
       : Map(NumInitBuckets), Data(Data) {}
 
+  bool hasMD() const { return static_cast<bool>(MDMap); }
-  bool hasMD() const { return MDMap; }
   MDMapT &MD() {
     if (!MDMap)
       MDMap.reset(new MDMapT);
